testkit/result.go

```diff
@@ -130,3 +130,42 @@ func (res *Result) CheckAt(cols []int, expected [][]interface{}) {
 	need := fmt.Sprintf("%s", expected)
 	res.require.Equal(need, got, res.comment)
 }
+
+// CheckContain checks whether the result contains the expected string
+func (res *Result) CheckContain(expected string) {
+	for _, row := range res.rows {
+		for _, colValue := range row {
+			if strings.Contains(colValue, expected) {
+				return
+			}
+		}
+	}
+	comment := fmt.Sprintf("the result doesn't contain the exepected %s", expected)
+	res.require.Equal(true, false, comment)
+}
+
+// CheckContainMore checks whether the result contains the expected string
+func (res *Result) CheckContainMore(expecteds []string) {
```

Comment:
IMHO the name is confusing... how about:
`CheckContainMore` -> `MultiCheckContain`
`CheckNotContainMore` -> `MultiCheckNotContain`

Also, please update the comments for those util functions.